### PR TITLE
Please review

### PR DESCRIPTION
**Pull request by AayushJoshiCCTECH**
Screenshot 2024-05-11 233532

### CODE REVIEW

File: Vox/Vox/src/Triangulation.cpp
Comment (vinayaksutar) [inline Vox/Vox/src/Triangulation.cpp:1]: No code?

File: Vox/Vox/src/CubeCreator.cpp
Comment (vinayaksutar) [inline Vox/Vox/src/CubeCreator.cpp:1]: Correct the file name

File: Vox/Vox/resources/sphere.stl
Comment (vinayaksutar) [inline Vox/Vox/resources/sphere.stl:1]: Remove this. This should not in git

File: Vox/Vox/resources/cube1.stl
Comment (vinayaksutar) [inline Vox/Vox/resources/cube1.stl:1]: Remove file

File: Vox/Vox/headers/Vox.h
Comment (vinayaksutar) [inline Vox/Vox/headers/Vox.h:1]: Code cleanup

File: Vox/Vox/headers/CubeCreator.h
Comment (vinayaksutar) [inline Vox/Vox/headers/CubeCreator.h:1]: Correct function names

**Review by vinayaksutar — CHANGES_REQUESTED** [review]
1. Code cleanup
2. Remove generated files and folders

File: Vox/Vox/src/Triangulation.cpp
Comment (AayushJoshiCCTECH) [reply Vox/Vox/src/Triangulation.cpp:1]: Instead of triangulation used GeomContainer. Needed to delete this file.

**Pull request closed without merging**